doc/api/api_changes_3.3/behaviour.rst

```diff
@@ -163,3 +163,9 @@ The default method used to format `.Slider` values has been changed to use a
 values are displayed with an appropriate number of significant digits even if
 they are much smaller or much bigger than 1.  To restore the old behavior,
 explicitly pass a "%1.2f" as the *valfmt* parameter to `.Slider`.
+
+Qt and wx backends no longer create a status bar by default
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The coordinates information is now displayed in the toolbar, consistently with
```

Comment:
It looks like for Qt we never created the status bar, we just injected an extra widget into it?

Reply:
We did -- just calling statusBar() was enough to create one implicitly.